src/foundation/components/Placeholder/index.tsx

```diff
@@ -0,0 +1,176 @@
+ *         margin-right: 2px;
+ *       }
+ *
+ *       .sendbird-place-holder__body--align-top__text__channel-name {
+ *         display: block;
+ *         max-width: calc(100% - 62px);
+ *         white-space: nowrap;
+ *         overflow: hidden;
+ *         text-overflow: ellipsis;
+ *       }
+ *     }
+ *
+ *     .sendbird-place-hlder__body--align-top__searching,
+ *     .sendbird-place-hlder__body--align-top__no-result
+ *     {
+ *       display: flex;
+ *       justify-content: center;
+ *     }
+ *   }
+ * }
+ * */
+// export default function PlaceHolder({
+//   className = '',
+//   type,
+//   iconSize,
+//   searchInString = '',
+//   retryToConnect,
+// }: PlaceHolderProps): ReactElement {
+//   const { stringSet } = useContext(LocalizationContext);
+//   return (
+//     <div className={[...(Array.isArray(className) ? className : [className]), 'sendbird-place-holder'].join(' ')}>
+//       {(type === PlaceHolderTypes.NO_RESULTS ||
+//         type === PlaceHolderTypes.SEARCH_IN ||
+//         type === PlaceHolderTypes.SEARCHING) && (
+//         <div className="sendbird-place-holder__body--align-top">
+//           {type === PlaceHolderTypes.SEARCH_IN && (
+//             <div className="sendbird-place-holder__body--align-top__text">
+//               <Label
+//                 className="sendbird-place-holder__body--align-top__text__search-in"
+//                 type={LabelTypography.BUTTON_2}
+//                 color={LabelColors.ONBACKGROUND_2}
+//               >
+//                 {stringSet.SEARCH_IN}
+//               </Label>
+//               <Label
+//                 className="sendbird-place-holder__body--align-top__text__channel-name"
+//                 type={LabelTypography.BUTTON_2}
+//                 color={LabelColors.PRIMARY}
+//               >
+//                 {`'${searchInString}`}
+//               </Label>
+//               <Label
+//                 className="sendbird-place-holder__body--align-top__text__quote"
+//                 type={LabelTypography.BUTTON_2}
+//                 color={LabelColors.PRIMARY}
+//               >
+//                 {"'"}
+//               </Label>
+//             </div>
+//           )}
+//           {type === PlaceHolderTypes.SEARCHING && (
+//             <Label
+//               className="sendbird-place-hlder__body--align-top__searching"
+//               type={LabelTypography.BODY_1}
+//               color={LabelColors.ONBACKGROUND_2}
+//             >
+//               {stringSet.SEARCHING}
+//             </Label>
+//           )}
+//           {type === PlaceHolderTypes.NO_RESULTS && (
+//             <Label
+//               className="sendbird-place-hlder__body--align-top__no-result"
+//               type={LabelTypography.BODY_1}
+//               color={LabelColors.ONBACKGROUND_2}
+//             >
+//               {stringSet.NO_SEARCHED_MESSAGE}
+//             </Label>
+//           )}
+//         </div>
+//       )}
+//     </div>
+//   );
+// }
```

Comment:
Should we keep this comment?

Reply:
I've left only the code for the part that needs to be implemented additionally.